pkg/operator/controllers/checker/checker_controller.go

```diff
@@ -37,7 +37,6 @@ func NewReconciler(log *logrus.Entry, maocli maoclient.Interface, arocli aroclie
 
 	if role == operator.RoleMaster {
 		checkers = append(checkers,
-			NewMachineChecker(log, maocli, arocli, role, isLocalDevelopmentMode),
```

Comment:
It looks like we no longer need `isLocalDevelopmentMode` argument in `NewReconciler`.